lib/fed_email_domain_downloader.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'csv'
+require 'faraday'
+require 'pry'
+require 'fileutils'
+
+class FedEmailDomainDownloader
+  attr_reader :destination
+
+  DOT_GOV_DOWNLOAD_PATH = 'https://raw.githubusercontent.com/cisagov/dotgov-data/main/current-federal.csv'
+
+  def initialize(destination: 'tmp/fed_download_path')
+    @destination = destination
+  end
+
+  def dot_gov_csv_path
```

Comment:
The name here would imply the method returns a path, but it returns the contents of the CSV. I'd suggest removing the suffix.

```suggestion
  def dot_gov_csv
```